parser/model/model.go

```diff
@@ -602,6 +602,8 @@ func (t TiFlashMode) String() string {
 type TiFlashReplicaInfo struct {
 	Count                 uint64
 	LocationLabels        []string
+	Ready                 bool
```

Comment:
Can you add some comments for `Ready` and `Available` ? These two states are confusing.